stores/suggestion_store.jsx

```diff
@@ -315,7 +316,7 @@ class SuggestionStore extends EventEmitter {
             this.emitSuggestionsChanged(id);
             break;
         case ActionTypes.SUGGESTION_COMPLETE_WORD:
-            if (this.areSuggestionsPending(id)) {
+            if (this.areSuggestionsPending(id) && !other.override) {
```

Comment:
override flag, when true allows `this.completeWord` to execute. Used when the user has finished typing before a suggestion is returned from the server